clang/test/CIR/CodeGen/temporaries.cpp

```diff
@@ -25,3 +25,9 @@ void f() {
 // CHECK-NEXT:   }
 // CHECK-NEXT:   cir.return
 // CHECK-NEXT: }
+
+const unsigned int n = 1234;
+const int &r = (const int&)n;
+
+//      CHECK: cir.global "private"  constant internal @_ZGR1r_ = #cir.int<1234> : !s32i
+// CHECK-NEXT: cir.global  external @r = #cir.global_view<@_ZGR1r_> : !cir.ptr<!s32i> {alignment = 8 : i64}
```

Comment:
Can you please add a LLVM check here? See `clang/test/CIR/CodeGen/abstract-cond.c` for an example. I want to make sure we are getting it right all the way, thanks!